### ISSUE TITLE
Missing styles with bootstrap 4

### ISSUE THREAD

**Issue by JakeGinnivan** [shakacode/bootstrap-loader#51]
I have been trying to get bootstrap 4 running in little test app I am putting together but am having issues. The container is hidden behind the nav.

https://github.com/JakeGinnivan/WikiThing/commit/9ec255bb254f8e6809ebca712263d3071bc69f78

As far as I can tell there are clearfix styles missing from the mixins, there may be other things missing as well. I cannot see what would be causing it and will probably just drop bootstrap 4 but thought id report my issue to see if someone knows why I am seeing what I am seeing.

**Comment by JakeGinnivan** [comment]
You should be able to just `npm start` on that repo to reproduce the issue

**Comment by justin808** [comment]
@JakeGinnivan Is there a chance that you can help @alexfedoseev and I debug this? We're about to ship our product, so we're slammed.

**Comment by JakeGinnivan** [comment]
Sure I can help, not sure where to go next though. This is not high priority so maybe when you guys get a chance to look at this project again just ping me and maybe we can nut it out. 

I spent an evening hitting a wall with it so have just gone back to bootstrap 3 for the moment.

**Comment by justin808** [comment]
@JakeGinnivan are you 100% sure the loader is the issue and it's just not bootstrap 4 which is still in Beta. I'm sort of planning on hitting this hard once bootstrap is at a RC.

**Comment by justin808** [comment]
I misspoke, "alpha" not "beta"...

**Comment by JakeGinnivan** [comment]
I'm not sure the loader is the issue.

But if I don't use the loader and just reference the .min.js file directly things seem to work fine. Let me have another go over the next day or two and give you a list of things I have tried. To be honest I forget the permutations I tried, I also pulled down the example project trying to see what is different. The actual styles injected into the browser seemed to be different in my project to the example but couldn't figure out why.. I'm happy to close this and wait for beta, just thought id open an issue to discuss or see if I am doing something stupid that I can't see.
